src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -163,7 +142,7 @@ const NavHeader = (props) => {
               })
             }>
             {!isFollow && isHeaderHide ? (
-              <ButtonFollow handleSetFollow={handleFollowTopic} />
+              <ButtonFollow handleSetFollow={onFollowButtonPress} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleSetFollow` prop of `ButtonFollow` has been updated to use `onFollowButtonPress` instead of `handleFollowTopic`. This change seems fine as long as the `onFollowButtonPress` function is passed as a prop from the parent component.
```diff
- <ButtonFollow handleSetFollow={handleFollowTopic} />
+ <ButtonFollow handleSetFollow={onFollowButtonPress} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->